### DISCUSSION TITLE
Please Add scroll volume for all taskbar panel.

### DISCUSSION THREAD

**Discussion by korenevskiy** [File-New-Project/EarTrumpet#707] [Ideas]
I'm tired of using the closed "volume2" project.
EarTrumpet is much cooler, except for one thing.
I still have to use Volume2.
You need to add a volume change by scrolling the wheel over the entire taskbar.

**Comment by vertigo220** [comment]
7++ Taskbar tweaker can do this and a whole lot more. I'd recommend checking it out.

**Comment by korenevskiy** [comment]
I got acquainted with 7++ Taskbar tweaker.
I still think that We need to add a volume change by scrolling the wheel across the entire width of the taskbar.
